Sources/Fuzzilli/Execution/REPRL.swift

```diff
@@ -93,11 +106,15 @@ public class REPRL: ComponentBase, ScriptRunner {
             // to execute this program. If we repeatedly fail to execute any program, we abort.
             if status < 0 {
                 logger.warning("Script execution failed: \(String(cString: reprl_get_last_error(reprlContext))). Retrying in 1 second...")
+                if fuzzer.config.diagnostics {
+                    // Log the buffer to disk
```

Comment:
Maybe remove this comment, since that only happens if the Storage module is active (and also since other users could react to this event)